docs/GERMLINE_SNV_README.md

```diff
@@ -121,6 +121,8 @@ For more information on the specific annotations, please see the documentation.
     - `peddy_html`: HTML metrics files from Peddy
     - `peddy_csv`: CSV metrics for het_check, ped_check, and sex_check from Peddy
     - `peddy_ped`: PED file with additional metrics information from Peddy
+    - `gatk_hardfiltered_vcf`: VCF containing the all genotyping variants with their hardfilters
```

Comment:
Does `vep_annotated_gatk_vcf` end up being a PASS + annotated version of this, or just annotated? If the latter, maybe we don't need this

Reply:
Oh yeah, I removed it from the WF as its not needed. Forgot to get it out of the READMEs